econml/metalearners.py

```diff
@@ -95,13 +107,14 @@ def marginal_effect(self, X):
         ----------
         X : matrix, shape (m × dₓ)
             Matrix of features for each sample.
+        T1: scaler
 
         Returns
         -------
         τ_hat : array-like, shape (m, )
```

Comment:
This looks wrong to me: the marginal effect should have shape (m, n_t-1), like it does for discrete treatments in DML